examples/hosts/app-integration/external-data/tests/index.tsx

```diff
@@ -31,26 +31,31 @@ export async function createContainerAndRenderInElement(element: HTMLDivElement)
 		const createResponse = await sessionStorageModelLoader.createDetached("1.0");
 		model = createResponse.model;
 
-		// Add a test task so we can see something.
-		model.taskList.addDraftTask("1", "testName", 3);
-
 		id = await createResponse.attach();
 	} else {
 		id = location.hash.slice(1);
 		model = await sessionStorageModelLoader.loadExisting(id);
 	}
 
-	// update the browser URL and the window title with the actual container ID
-	// eslint-disable-next-line require-atomic-updates
-	location.hash = id;
-	document.title = id;
+	model.baseDocument.addTaskList({ externalTaskListId: "task-list-test" });
+
+	const taskList = model.baseDocument.getTaskList("task-list-test");
+	// Add a test task so we can see something.
+	if (taskList !== undefined) {
+		taskList.addDraftTask("1", "testName", 3);
 
-	// Render it
-	ReactDOM.render(<TaskListView taskList={model.taskList} />, element);
+		// update the browser URL and the window title with the actual container ID
+		// eslint-disable-next-line require-atomic-updates
```

Comment:
The docs for this rule suggest to me that it's not needed here anymore.

Reply:
It still complains when I remove it, so leaving the rule in.